freud/data.py

```diff
@@ -68,9 +68,24 @@ def generate_system(self, num_replicas=1, scale=1, sigma_noise=0, seed=None):
 
         Note:
             Positions are generated in the order of the instance's
-            ``basis_positions``. The first :math:`N_{replica}`
-            positions come from the first basis position, the next
-            :math:`N_{replica}` the second, etc.
+            ``basis_positions``. The first :math:`N_{replica}` positions come
+            from the first basis position, the next :math:`N_{replica}` the
+            second, etc. A NumPy one-liner that generates the list of indices is
+            ``np.repeat(np.arange(len(uc.basis_positions)), np.product(N))``.
+
+        Below is an example of expanding basis position properties such as type
+        to a replicated lattice.
+
+        .. code-block:: python
```

Comment:
I'd write this as an example that will be recognized by the docs and will be doctested. Example of example writing: https://github.com/glotzerlab/freud/blob/3daeba88bb8e6e11973694a78f6eb652978b7f16/freud/box.pyx#L397-L406

Reply:
I forgot freud did doc testing.